recipes/perl-class-inspector/recipe.yaml

```
@@ -0,0 +1,46 @@
+schema_version: 1
+
+context:
+  name: perl-class-inspector
+  version: 1.36
+
+package:
+  name: ${{ name }}
+  version: ${{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/P/PL/PLICEASE/Class-Inspector-${{ version }}.tar.gz
+  sha256: cc295d23a472687c24489d58226ead23b9fdc2588e522f0b5f0747741700694e
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - perl Makefile.PL INSTALLDIRS=site NO_PERLLOCAL=1 NO_PACKLIST=1
+    - make
+    - make test
+    - make install
+
+requirements:
+  build:
+    - make
+  host:
+    - perl=5.32
+  run:
+    - perl>=5.32
```

```suggestion
  host:
    - perl
  run:
    - perl
```
We don't have a `noarch: perl` equivalent to `noarch: python`, so I think even for `noarch: generic` packages, we can't just set minimum version.  We need to update the `perl` feedstock to fill in all the newer versions we are missing, and then we can update the feedstock pinnings to cover all supported Perl versions.  And eventually it would be great to properly implement a `noarch: perl` so that we don't have to build separate `noarch` packages for each version of Perl.

---

This should apply to all recipes as well - I meant to apply to the first one like the other comments!